packages/components/src/FormField/FormField.css

```diff
@@ -99,7 +99,16 @@ select.formField {
 .inline {
   display: inline-block;
   width: auto;
-  vertical-align: -moz-middle-with-baseline;
```

Comment:
I took a quick look at Firefox and I was able to just remove this line altogether and got the same result. Just wondering if we need the `vertical-align: baseline` or if we can just remove the `vertical-align` altogether?

Reply:
`baseline` is the initial value for `vertical-align`, and it's not an inherited property, so that could be removed. I like explicitly setting it – even if it's the default – if it's important enough to the goal of the class (the next person to see this code can see that explicitly matching the baseline is the intent), but I suppose that does make for extra CSS bloat.

Reply:
Removed the unnecessary `vertical-align` statement along with another cleanup